automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/RuntimeFixtureSpec.scala

```diff
@@ -119,26 +119,29 @@ abstract class RuntimeFixtureSpec
     super.beforeAll()
     logger.info("beforeAll")
 
-    sys.props.get(gpallocProjectKey) match {
+    sys.props.get(workspaceNamespaceKey) match {
```

Comment:
Also looks like an incomplete match -- I'd probably again suggest removing this and putting the `gpallocErrorPrefix ` in the `googleProjectKey` prop